judoscale-ruby/lib/judoscale/report.rb

```diff
@@ -12,7 +12,7 @@ def initialize(adapters, config, metrics = [])
 
     def as_json
       {
-        dyno: config.runtime_container,
+        container: config.runtime_container,
```

Comment:
Going to need to make a slight back-end change on V3 to accommodate this one before this PR gets released!